engine/src/main_loop.cpp

```diff
@@ -851,10 +848,10 @@ void createObjects( std::vector< std::string > &fighter0name,
                             dat->clear();
                         }
                         fighter0mods.push_back( modifications = game_options.getCallsign( squadnum ) );
-                        if ( squadnum == 0 && global_username.length() ) {
-                            fighter0mods.back() = global_username;
-                            modifications = global_username;
-                        }
+//                        if ( squadnum == 0 && global_username.length() ) {
+//                            fighter0mods.back() = global_username;
+//                            modifications = global_username;
+//                        }
```

Comment:
In the other 99 cases, I'd delete and let the CM keep a record of it. However, in this case, I left it in because I didn't understand this code fully. I don't think we need it but prefer to keep a record in place for the person investigating the resulting obscure bug. 
As we refactor, it'll be much easier to delete. Especially when we understand the role of squadrons and modifications in the game.